diff.html

```diff
@@ -254,7 +254,7 @@ <h5>Warframe Drop Data - Compare Versions</h5>
     $(document).ready(function() {
         let now = new Date().getTime()
 
-        $.getJSON(`/data/builds/builds.json?${now}`, function(builds) {
+        $.getJSON(`./data/builds/builds.json?${now}`, function(builds) {
```

Comment:
why are we using it as a query variable key instead of value? specifically, something like.... `./data/builds/builds.json?v=${now}`

This is the cache-busting used by places like StackOverflow, which I'm assuming is what we're going for.

Reply:
That doesn't really matter though. It's for cache busting and it works either way, the URL only has to change in some way.

Reply:
Okiedokie